libnullpay/src/lib.rs

```diff
@@ -23,9 +23,10 @@ use std::ffi::CString;
 
 #[no_mangle]
 pub extern fn nullpay_init() -> ErrorCode {
-    if let Err(err) = utils::logger::LibnullpayLogger::init() {
```

Comment:
I suggest to use platform specific directive instead of commenting code

Reply:
Also please add comment and reference to ticket/bug in Jira about resolving mobile platform problem

Reply:
I added directives.